### ISSUE TITLE
Cannot find module 'resolve-from'

### ISSUE THREAD

**Issue by ajay28kumar** [preactjs/preact-cli#1410]
**Do you want to request a _feature_ or report a _bug_?**
bug
**What is the current behaviour?**
After running `npm run dev` getting error `Cannot find module 'resolve-from'`.
![Screenshot 2020-09-11 at 7 07 03 PM](https://user-images.githubusercontent.com/16662534/92932047-fce39600-f461-11ea-98d0-d5dbda323d4e.png)

**If the current behaviour is a bug, please provide the steps to reproduce.**
I created a new project using `npx preact-cli create material preact-project` and my `package.json` looks like this.
![Screenshot 2020-09-11 at 7 09 47 PM](https://user-images.githubusercontent.com/16662534/92932323-5f3c9680-f462-11ea-8dcf-b72343736145.png)

Then to upgrade `preact-cli` and `preact-js` I've deleted `node_modules` and manually changed my `package.json`. Now it looks likes this:-
![Screenshot 2020-09-11 at 7 11 58 PM](https://user-images.githubusercontent.com/16662534/92932547-ac206d00-f462-11ea-8200-c869b330026d.png)

After this running `npm run dev` is throwing error.

**What is the expected behaviour?**
Ideally, it should have been running Smoothly.

**If this is a feature request, what is motivation or use case for changing the behaviour?**
Library Upgrade because the latest library has automatic code-splitting 

**Please mention other relevant information.**
There should be proper documentation for upgrading the `preact-cli`
Please paste the results of `preact info` here.
```

Environment Info:
  System:
    OS: macOS 10.15.2
    CPU: (4) x64 Intel(R) Core(TM) i5-5350U CPU @ 1.80GHz
  Binaries:
    Node: 12.14.1 - /usr/local/bin/node
    Yarn: 1.19.0 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
  Browsers:
    Chrome: 85.0.4183.102
    Safari: 13.0.4
  npmPackages:
    preact: ^10.4.8 => 10.4.8 
    preact-cli: ^3.0.1 => 3.0.1 
    preact-render-to-string: ^5.1.10 => 5.1.10 
    preact-router: ^3.2.1 => 3.2.1 
  npmGlobalPackages:
    preact-cli: 3.0.1

```

**Comment by RyanChristian4427** [comment]
The `resolve-from` issue was noted and fixed with #1382. You'll just have to wait for a new release on that front.

**Comment by RyanChristian4427** [comment]
Fix should be released under v3.0.2. Can you see if the issue is still reproducible after an update?
